host/tests/dsClasses/dsVideoPort.py

```diff
@@ -585,6 +585,53 @@ def resetOutputToSDR(self,video_port:int, port_index:int=0):
 
         result = self.utMenu.select(self.testSuite, "ResetOutputToSDR", promptWithAnswers)
 
+    def setAllmMode(self, video_port:int,  port_index:int=0, mode:str="Disable"):
+        """
+        Resets the video output to Standard Dynamic Range (SDR) for the specified video port.
+
+        This method allows you to This method allows you to enables or disables the Auto Low Latency Mode (ALLM) 
+        for a HDMI output video port on source devices, as per the HDMI 2.1 specification.
+
+
+        Args:
+            video_port (int): The enumeration value representing the video port.
+                            Refer to the dsVideoPortType enum for valid options.
+            port_index (int, optional): The index of the specific port to reset. Defaults to 0.
+            mode (str, optional) : ALLM mode, defaults to "Disable"
+
+        Returns:
+            None
+
+        Example:
+            setAllmMode(video_port=dsVIDEOPORT_TYPE_HDMI, port_index=0, mode:str="Disable")
+        """
+        promptWithAnswers = [
+            {
+                "query_type": "list",
+                "query": "Select the Video Port",
+                "input": "dsVIDEOPORT_TYPE_HDMI"
+            },
+            {
+                "query_type": "direct",
+                "query": "Select the Video Port Index[0-9]:",
+                "input": "0"
+            },
+            {
+                "query_type": "list",
+                "query": "Choose ALLM mode to be Enable/Disable:",
+                "input": "Disable"
+            }
+        ]
+
+        # Convert input arguments to strings and update the prompts
+        promptWithAnswers[0]["input"] = str(video_port)
```

Comment:
could you please directly assign strings to input like below.
"input": str(video_port)
"input": str(port_index)
"input": str(mode)